src/PKSim.Core/Commands/UpdateParameterValueOriginCommand.cs

```diff
@@ -0,0 +1,47 @@
+﻿using OSPSuite.Core.Commands;
+using OSPSuite.Core.Domain;
+using OSPSuite.Utility.Extensions;
+using PKSim.Core.Model;
+
+namespace PKSim.Core.Commands
+{
+   public class UpdateParameterValueOriginCommand : PKSimMacroCommand
```

Comment:
I have encapsulated the action in a command for now. That means that updating the value origin will result in an entry in the history